### DISCUSSION TITLE
Information transparency on fields in the wallet display

### DISCUSSION THREAD

**Discussion by y04nqt** [polymorpher/one-wallet#96] [Q&A]
Questions:
1. What does 'Expires In' mean to me? This seems to be a worrying point of transferring more currency into the wallet with an adequate description here.
2. Daily Limit? Is this meaning buying NFTs or transferring out of the wallet per day? I assume it only means to send somewhere else. We should reassure users of this action variable.
3. What is the relayer and lock feature? More context and info should be supplied somewhere. Maybe in a /FAQ page to avoid making extra features that may be a higher lift.

**Comment by polymorpher** [comment]
Sorry for the confusion. There will be tutorials, guides, and documentations. We might add a FAQ page later.  We may also simplify or remove some terms.

1. As of now, "Expires In" means the date when the wallet reaches its end of life. The only thing you can do with the wallet at that time is to recover the funds to a recovery address. We are still evaluating the trade-offs on whether we should let the wallet auto-extend its lifespan.
2. Daily Limit is fund transfer limit per day.
3. The lock icon sets the password for the relayer. Some descriptions of what a relayer does is provided there.
